free-tunes/Service Layer/CustomTypes.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  CustomTypes.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/03/29.
+//
+
+import Foundation
+
+typealias FetchMusicResult = Result<TracksModel, Error>
```

Comment:
Hmm not a huge fan of having a file where you keep all the custom types, might be best to have this above the `protocol`.